File: C++ solutions/1347. Minimum Number of Steps to Make Two Strings Anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        map<char,int>m;
        for(int i=0;i<s.size();i++) m[s[i]]++;
        for(int i=0;i<t.size();i++){
            if(m[t[i]]){
                m[t[i]]--;
            }
        }
        int ans=0;
        for(auto k:m) ans+=k.second;
        return ans;
    }
};

File: JavaScript solutions/1347. Minimum Number of Steps to Make Two Strings Anagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var minSteps = function (s, t) {
  const m = new Map();

  for (let i = 0; i < s.length; i++) {
    if (!m.has(s[i])) {
      m.set(s[i], 1);
    } else {
      m.set(s[i], m.get(s[i]) + 1);
    }
  }

  for (let i = 0; i < t.length; i++) {
    if (m.has(t[i]) && m.get(t[i]) > 0) {
      m.set(t[i], m.get(t[i]) - 1);
    }
  }

  let ans = 0;
  for (const [key, value] of m.entries()) {
    ans += value;
  }
  return ans;
};
